pkg/executables/kubectl.go

```diff
@@ -690,6 +690,18 @@ func (k *Kubectl) ValidateEKSAClustersCRD(ctx context.Context, cluster *types.Cl
 	return nil
 }
 
+func (k *Kubectl) RestartCiliumDaemonset(ctx context.Context, kubeconfig string) error {
```

Comment:
This should be a `DaemonSetRolloutRestart call` instead of a reference to Cilium in kubectl

Reply:
Agreed, will pass in the daemonset name instead of hardcoding it here